supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift-container-storage.adoc

```diff
@@ -0,0 +1,147 @@
+[[openshift-container-storage-conventions]]
+
+[discrete]
+[[backing-store]]
+==== backing store (noun)
+*Description*: A type of storage resource for Multicloud Object Gateway, which is used to store data, for example, from RADOS gateway (RGW), Amazon Web Services S3, Azure Blob Storage, IBM Cloud Object Storage. 
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[container-storage-interface]]
+==== Container Storage Interface (noun)
+*Description*: The Container Storage Interface (CSI) is a standard for exposing arbitrary block and file storage systems to containerized workloads on Container Orchestration Systems like Kubernetes, and in particular Red Hat OpenShift Container Platform. This allows OpenShift Container Platform to consume storage from third-party storage providers that implement the CSI interface as persistent storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[multicloud-object-gateway]]
+==== Multicloud Object Gateway (noun)
+*Description*: Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform you can use to start with a small storage service and then scale according to your requirements on-premises, in multiple clusters, and with cloud-native storage.
+
+Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform that you can use to start with a small storage service and then scale according to your requirements on-premise, in multiple clusters, and with cloud-native storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-bucket]]
+==== namespace bucket (noun)
+*Description*: Namespace buckets enable connecting data repositories on different providers together so that the interaction with data is possible through a single unified view. Data written to namespace buckets are kept plain and not deduped, compressed, or encrypted. The data can be consumed directly from the repository.
```

Comment:
To remove passives here, you could do something like this:

`Namespace buckets enable connecting data repositories on different providers together so that the interaction with data is possible through a single unified view. Data that you write to namespace buckets are plain and not deduped, compressed, or encrypted. Users (or some other component) can consume the data directly from the repository.`

For the final line though, it's necessary to know who or what consumes the data before making this active. I also suggested in there to remove the passive 'are kept' because it's vague as well as passive and I guess it's OK to just say 'are plain and not deduped'.

Reply:
I have left the last sentence 'as is' but have made the other suggested changes.